### DISCUSSION TITLE
Welcome to NFT-Metadata Discussions!

### DISCUSSION THREAD

**Discussion by Yagi-Michael** [NFT-DAO/NFT-Metadata#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stephen-rowan** [comment]
Hello Everyone, 

My name is Stephen Whitenstall. I am a Philosophy Graduate presently moving into an MA (this year hopefully) with an interest in knowledge representation and the philosophical implications of Blockchain technologies. Like many others I have been drawn to the unique nature of the Cardano project in it's endeavor to unite finance, governance and communities in a decentralized, distributed blockchain.

Following a discussion on metadata in the Discord server of NFT DAO (the  Comprehensive NFT Framework Collaboration) I have on-boarded here to assist in using github (and any other tools), to document the dynamic ideas and meetings that are rapidly arising from NFT DAO and to participate in this exciting experiment !

Regards Stephen
